docs/quick-starts/framework/ruby/README.mdx

```diff
@@ -0,0 +1,43 @@
+---
+slug: /quick-starts/ruby
```

Comment:
Do we have ruby logo?

Reply:
no, i'll add it later

Reply:
https://github.com/logto-io/docs/pull/696